index.html

```diff
@@ -0,0 +1,86 @@
+  <link rel="stylesheet" href="./styles/nav-bar.css">
+  <link rel="stylesheet" href="./styles/styles.css">
+  <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap" rel="stylesheet">
+  <title>Global Training Courses - Home</title>
+</head>
+<body>
+  <header>
+    <nav class="nav-mobile">
+      <img class="menu-button" id="menu-button" src="./assets/hamburger-menu.svg" alt="menu">
+      <div class="menu" id="menu">
+        <div>
+          <img id="close-button" src="./assets/close.png" alt="close">
+        </div>
+        <ul>
+          <li><a href="about.html">About Us</a></li>
+          <li><a href="#topics">Topics</a></li>
+          <li><a href="#courses">Courses</a></li>
+          <li><a href="#">Log In</a></li>
+          <li><a href="#">Sign Up</a></li>
+        </ul>
+      </div>
+    </nav>
+    <nav class="nav-desktop">
+      <div class="black-bar">
+        <div>
+          <img src="./assets/facebook.png" alt="facebook-logo">
+        </div>
+        <div>
+          <img src="./assets/twitter.png" alt="twitter-logo">
+        </div>
+        <span>English</span>
+        <span>My Page</span>
+        <span>Log Out</span>
+      </div>
+      <ul>
+        <li><a href="about.html">About Us</a></li>
+        <li><a href="#topics">Topics</a></li>
+        <li><a href="#courses">Courses</a></li>
+        <li><a href="#">Log In</a></li>
+        <li><a href="#">Sign Up</a></li>
+      </ul>
+    </nav>
+    <div class="title-container">
+      <h1>GLOBAL TRAINING COURSES</h1>
+      <p>A swiftly evolving business landscape requires an equally dynamic workforce. One-dimensional skill sets are no longer the norm, and not an option. To possess a blend of people, process and technology know-how makes a complete professional.
+      <br><br>Give your career a boost by mastering people and process expertise in areas like communication, critical thinking, finance, leadership, negotiation and problem-solving.</p>
+    </div>
+  </header>
+  <main>
+    <section class="topics" id="topics">
+      <h3 class="topics-title">Topics</h3>
+      <div class="border-topic"></div>
+      <div class="topics-container" id="topics-container">
+      </div>
+      <button>Join Global Training Courses</button>
+    </section>
+    <section class="courses-section" id="courses">
+      <h3>Courses</h3>
+      <div class="border-topic"></div>
+      <div class="courses-container" id="courses-container">
+      </div>
+      <button class="more-button" id="more-button">More</button>
+    </section>
+  </main>
+  <footer>
+    <div>
+      <h4>GLOBAL TRAINING COURSES</h4>
+    </div>
+    <div>
+      <p>© 2022 Global Training Courses LLC. All Rights Reserved.</p>
+    </div>
+  </footer>
```

Comment:
- [ ] You missed `patner section` and also your footer is not currently looking like the project design
 your design :point_down: 
     
![Screenshot from 2022-04-24 00-43-34](https://user-images.githubusercontent.com/66097371/164949727-f97b6070-67b6-4263-9929-79d06ca35cf9.png)

project design :point_down: 
![Screenshot from 2022-04-24 00-46-11](https://user-images.githubusercontent.com/66097371/164949756-6d910db1-3116-4a0a-a17b-cf93a4badae2.png)